### DISCUSSION TITLE
Multiple selection

### DISCUSSION THREAD

**Discussion by thistlillo** [mikekazakov/nimble-commander#432] [Ideas]
The possibility to select multiple files here and there is very convenient, but very dangerous as well.

I think it would be a nice addition:
- a better way to signal that multiple items are selected
- a hotkey for deselecting all the files that are selected.

**Comment by mikekazakov** [comment]
Well NC highlights the selected items and shows the count with the combined size.
The hotkey already exists and is described in https://github.com/mikekazakov/nimble-commander/blob/main/Docs/Help.md#selection
